XMonad/Prelude.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE BangPatterns #-}
+--------------------------------------------------------------------
+-- |
+-- Module      :  XMonad.Prelude
+-- Copyright   :  slotThe <soliditsallgood@mailbox.org>
+-- License     :  BSD3-style (see LICENSE)
+--
+-- Maintainer  :  slotThe <soliditsallgood@mailbox.org>
+--
+-- Utility functions and re-exports for a more ergonomic developing
+-- experience.  Users themselves will not find much use here.
+--
+--------------------------------------------------------------------
+module XMonad.Prelude (
+    module Exports,
+    fi,
+    chunksOf,
+    (.:),
+    (!?),
+) where
+
+import Control.Applicative as Exports
+import Control.Monad       as Exports
+import Data.Bool           as Exports
+import Data.Char           as Exports
+import Data.Foldable       as Exports
+import Data.Function       as Exports
+import Data.Functor        as Exports
+import Data.List           as Exports
+import Data.Maybe          as Exports
+import Data.Monoid         as Exports
```

Comment:
I'm more generally worried about re-exporting too much for the same reason as above — the more reexports, the greater the chance of name collisions.